src/requests/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const ContentfulSigningHeader = {
```

Comment:
I don't get "signing" in the name. Maybe this should be "XContentfulHeader" enum?

Reply:
The idea was that we use these headers for the signing process :D But yeah, if it's clearer will change it to a more generic name